src/vivarium_ciff_sam/constants/data_values.py

```diff
@@ -96,6 +96,10 @@ class __Wasting(NamedTuple):
         name='value'
     )
 
+    DIARRHEA_DURATION_VICIOUS_CYCLE: Tuple = (
```

Comment:
are we expecting this to fix the duration of diarrhea issue?

Reply:
yes that's the idea. if we calibrate the rr to the actual duration observed in the sim it should be more coherent.